cookiecutter.json

```diff
@@ -1,5 +1,5 @@
 {
     "project_name": "Neuro Project",
-    "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '-').replace('_', '-') }}",
-    "code_directory": "modules"
+    "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "code_directory": "code"
```

Comment:
I remember Bender was against `/project/code` because there was a pip package `code`, but I [can't find it](https://pypi.org/search/?q=code).
I vote for `code` if there's no collision, or C-like naming `src`. But not `modules` because it's similar to `models`.

Reply:
I also do like `src` more than modules or code. Thanks for the suggestion!
Changed in 3524734